src/core/checkedint.d

```diff
@@ -682,10 +682,27 @@ unittest
 
 /// ditto
 pragma(inline, true)
+ulong mulu(uint x, ulong y, ref bool overflow)
+{
+    ulong r = x * y;
+    if (y >> 32 && x && (r / x) != y)
```

Comment:
I agree. But redoing the coverage analyzer is a longer term goal. In the meantime, let's just get those 3 clauses covered. It's important because overflow detection is an often misunderstood and misprogrammed issue. Having good coverage will inure us from someone "fixing" it in the future.